source/index.html

```diff
@@ -11,7 +11,7 @@
         <div id="mount">
             <div class="player-score-list">
                 <div class="score-container">
-                    <div id="p1-score">12</div>
+                    <div id="p1-score"></div>
```

Comment:
Thank you for fixing that; it's been bothering me when you see a little 12 pop up in the corner while the page loads. :x